docs/deploy-cephfs.md

```diff
@@ -223,6 +225,29 @@ The Helm chart is located in `charts/ceph-csi-cephfs`.
 
 [See the Helm chart readme for installation instructions.](../charts/ceph-csi-cephfs/README.md)
 
+## Read Affinity using crush locations for CephFS subvolumes
+
+Ceph CSI supports mounting CephFS subvolumes with kernel mount options
+`"read_from_replica=localize,crush_location=type1:value1|type2:value2"` to
+allow serving reads from the most local OSD (according to OSD locations as
+defined in the CRUSH map).
+Refer [krbd-options](https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options)
+for more details.
```

Comment:
This points to krbd mount options, 
These read affinity mount options are not documented for cephfs kernel mounter. I'll ask for it to be documented at https://docs.ceph.com/en/latest/man/8/mount.ceph/#options. 

Let's remove these two lines for now.

Reply:
Done